frontends/ol-util/src/components/dnd.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useMemo, useState } from "react"
```

Comment:
Should this file be SortableList.tsx or similar?

Reply:
Maybe... I usually name component files after their default export,  but in this case there are two named exports `{ SortableItem, SortableList }`... I guess List is the main component, though.